server/novaboot-shell

```diff
@@ -188,7 +225,7 @@ main() {
 	# Commands allowed at any time
 	"") locked $0 default;;
 	"console") locked $0 console;;
-	"get-config") read_config && echo -n "${target_config}"; exit;;
+	"get-config") read_config && get_nfsroot && echo "--nfsroot=$NB_NFSROOT" && echo -n "${target_config}"; exit;;
```

Comment:
Move this code to a new function `get_config`.

If the administrator disables the unfsd service somehow, the `--nfsroot` options should not be sent to the client at all.